kedro-docker/README.md

```diff
@@ -1,8 +1,8 @@
 # Kedro-Docker
 
-[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Python Version](https://img.shields.io/badge/python-3.6%20%7C%203.7%20%7C%203.8-blue.svg)](https://pypi.org/project/kedro-docker/)
+[![Python Version](https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue.svg)](https://pypi.org/project/kedro-telemetry/)
```

Comment:
Looks like it should be a `kedro-docker` link here? It links to telemetry now.

Reply:
Good spot! Silly copy-paste from my side.